modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -2082,6 +2091,15 @@ components:
                     type: boolean
                 verticalAlignment:
                     type: string
+                viewportRowConfiguration:
```

Comment:
I would use an array (maybe called viewportRowDefinitions), and include the identifier (landscapeMobile, portraitMobile, tablet) as an additional attribute of the referenced type.